packages/docs/content/utilities/ring.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Ring
+---
+
+import { Button } from '@cmsgov/design-system';
+
+Use the following utilities to apply focus styles to custom UI components.
+
+## Examples
+
+### Focus rings
+
+**Focus**
+
+Use the `.focus:ring` utility to conditionally apply focus styles to your component when that component has focus. Focus will be applied regardless of keyboard or mouse interaction.
+
+Click or tab into the following example to see this utility in action.
+
+<EmbeddedExample>
+  <p tabIndex="0" class="focus:ring ds-u-border--1 ds-u-padding--2">Focus state example</p>
```

Comment:
I struggled to create real world examples to showcase these utils. All the focussable elements in the DS already have focus styles baked in, so it was difficult showcasing the different behaviors of each utility.

Reply:
I'll try to think of a custom element that has been produced in downstream products

Reply:
Still not a real-world example, but I was looking through possible form elements, and I thought this might be cool:
```html
<input type="color" name="color-focus" value="#ff0000" class="focus:ring ds-u-radius" />
```